src/commands/paranet/start-paranet-sync-commands.js

```diff
@@ -0,0 +1,112 @@
+import Command from '../command.js';
+import { ERROR_TYPE, PARANET_SYNC_FREQUENCY_MILLS } from '../../constants/constants.js';
+
+class StartParanetSyncCommands extends Command {
+    constructor(ctx) {
+        super(ctx);
+        this.commandExecutor = ctx.commandExecutor;
+        this.ualService = ctx.ualService;
+        this.blockchainModuleManager = ctx.blockchainModuleManager;
+        this.repositoryModuleManager = ctx.repositoryModuleManager;
+
+        this.errorType = ERROR_TYPE.PARANET.START_PARANET_SYNC_ERROR;
+    }
+
+    async execute() {
+        const operationId = this.operationIdService.generateId();
+
+        this.logger.info(
+            `Paranet sync: Starting Paranet sync command for operation id: ${operationId}`,
+        );
+
+        await this.commandExecutor.delete('paranetSyncCommand');
+
+        const promises = [];
+        this.config.assetSync?.syncParanets.forEach(async (paranetId) => {
+            if (!this.ualService.isUal(paranetId)) {
```

Comment:
This check is performed during paranet initialization in ot-node.js it can be removed from here